bridge/ApiRequestHandler.cpp

```diff
@@ -71,6 +73,9 @@ httpd::Response ApiRequestHandler::callEndpointAction(RequestLogger& requestLogg
         (request._method == httpd::Method::POST || request._method == httpd::Method::DELETE))
         return processBarbellAction(this, requestLogger, request, token);
 
+    if (utils::StringTokenizer::isEqual(token, "probing") && request._method == httpd::Method::GET)
```

Comment:
Can we find a better name for the endpoint? The purpose is to get ice credentials for making a STUN RTT probe.

Reply:
Simply "probecandidates" maybe?

Reply:
or ice-candidates. That is what we are retrieving.

Reply:
I tried to distinguish between real ICE candidates and "simplified" ones, intended for probing only.

Reply:
Changed to "ice-candidates"